services/slugify.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * @adonisjs/lucid-slugify
+ *
+ * (c) AdonisJS
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import app from '@adonisjs/core/services/app'
+import { SlugifyDecorator } from '../src/types.js'
+
+let slugify: SlugifyDecorator
+
+await app.booted(async () => {
+  slugify = await app.container.make('slugify.decorator')
+})
```

Comment:
not sure about this.

in fact, here we're making a container service for the decorator. This container service can be imported and used like this: 

```ts
import slugify from '@adonisjs/lucid-slugify/decorator'

class Post {
  @slugify({ .. })
  public slug: string
}
```

it works, but I think it's a bit weird because so far we've never used a container service for this kind of thing

it's the only solution I have found because the decorator needs `SlugifyManager`, and the IoC manages its instantiation. 

if a better solution is desirable, please let me know

Reply:
Yeah, that seems awful. The decorators shouldn't be turned into container services. Seems like, we will need a bit of refactoring within the codebase. Lemme give that a try